.github/workflows/ci.yml

```
@@ -622,6 +622,36 @@ jobs:
             bash -euo pipefail -c \
               "./build_tools/cmake/build_riscv.sh && ./build_tools/cmake/test_riscv64.sh"
 
+  emscripten:
+    needs: [setup, build_all]
+    if: needs.setup.outputs.should-run == 'true'
```

@GMNGeoffrey you wanted this to start as post-submit only? Should that go in a different workflow file, or is there a preferred way to configure a step as only running on postsubmit?

---

Yeah we can add a condition to the if for now

```suggestion
  emscripten:
    needs: [setup, build_all]
    if: needs.setup.outputs.should-run == 'true' and needs.setup.outputs.ci-stage == 'postsubmit'
```

And then actually add that output to setup :-) (stupid GitHub won't let me comment on far-away lines).

The other option is making should-run a list of a jobs that should run. It's something I've been thinking about, but it seems like a bit of a pain right now because then it's yet-another place that has to know about the job names. I'm still pondering some sort of generation for this stuff to avoid that duplication